DragaliaAPI.Shared/MasterAsset/Models/Missions/DrillMissionGroup.cs

```diff
@@ -7,11 +9,11 @@ public record DrillMissionGroup(
     string TextName,
     string TextNameShort,
     int Step,
-    int TextTitle,
+    string TextTitle,
     string TextDetail,
     EntityTypes UnlockEntityType1,
     int UnlockEntityId1,
     int UnlockEntityQuantity1,
-    string StartDate,
-    string EndDate
+    [property: JsonConverter(typeof(MasterAssetDateTimeOffsetConverter))] DateTimeOffset StartDate,
```

Comment:
Can't remember if you've explained before -- do we need this if adding the converter in MasterAssetJsonOptions?

Reply:
nope, we dont - resolved